lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -147,6 +147,15 @@
   groups:
     description:
       - List of additional group names and/or IDs that the container process will run as.
+  healthcheck:
+    type: dict
+    description:
+      - Configure a check that's run to determine whether container is healthy
+      - 'I(test) - Test to perform to determine container health.'
+      - 'I(interval) - The time to wait between check.'
```

Comment:
Here (or somewhere else) you should describe in which format time can be specified.